### PR TITLE
New devices: Trust ZLED-TUNE9, DIYRuZ contact, Lonsonho Plug

### PR DESCRIPTION
**Pull request by kirovilya**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -1384,6 +1384,15 @@ const devices = [
             };
         },
     },
+    {
+        zigbeeModel: ['DIYRuZ_magnet'],
+        model: 'DIYRuZ_magnet',
+        vendor: 'DIYRuZ',
+        description: 'DIYRuZ contact sensor',
```
Comment (Koenkk) [inline devices.js]: Do you have a link on how to build this device?

Comment (kirovilya) [reply devices.js]: yes. I am forgot... fixed

File: devices.js [hunk 2]
```diff
@@ -5746,6 +5762,24 @@ const devices = [
             execute(device, actions, callback);
         },
     },
+
+    // Lonsonho
+    {
+        zigbeeModel: ['Plug_01'],
+        model: 'Zigbee plug',
+        vendor: 'Lonsonho',
```
Comment (Koenkk) [inline devices.js]: https://nl.aliexpress.com/item/32991703204.html is this the device?

Comment (kirovilya) [reply devices.js]: no. this https://ru.aliexpress.com/item/4000116784070.html

**Pull request merged**